spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -45,6 +45,24 @@ def expect_retryable_error(error_class)
           expect(Form526JobStatus.last.status).to eq 'success'
         end
       end
+
+      context 'with an MAS-related diagnostic code' do
+        let(:submission) do
+          create(:form526_submission,
+                 :non_rrd_with_mas_diagnostic_code,
+                 user_uuid: user.uuid,
+                 auth_headers_json: auth_headers.to_json,
+                 saved_claim_id: saved_claim.id)
+        end
+
+        it 'sends an email for tracking purposes' do
+          VCR.use_cassette('evss/disability_compensation_form/submit_form_v2') do
+            subject.perform_async(submission.id)
+            described_class.drain
+            expect(ActionMailer::Base.deliveries.last.subject).to eq 'MA claim - 6847'
+          end
+        end
+      end
     end
```

Comment:
Probably add a test with multiple MAS-related DCs and check that a notification is not sent.

Reply:
done